content/applications/finance/expenses.rst

```diff
@@ -4,231 +4,544 @@
+   :align: center
+   :alt: Submit the report to the manager.
 
-How to submit expenses to managers
-==================================
+.. Note::
+   The status of each report is shown in the :guilabel:`Status` column on the far right. If the
+   :guilabel:`Status` column is not visible, click the :guilabel:`⋮ (additional options)` icon at
+   the end of the row, and check the box next to :guilabel:`Status`.
 
-When you are ready to submit your expenses to your manager
-(e.g. at the end of a business trip, or once a month),
-go to the menu :menuselection:`My Expenses --> Expenses to Submit`. Select all expenses
-from the list view and click on :menuselection:`Action --> Submit to Manager`.
-Save the newly created expense report (i.e. set of expenses),
-and wait for your manager to approve it.
+Click on a report to open it, then click :guilabel:`Submit To Manager`.
 
-.. image:: expenses/submit_03.png
+.. image:: expenses/submit-to-manager.png
    :align: center
+   :alt: Submit the report to the manager.
 
-You can also submit expenses one by one from the *Submit to Manager*
-button on the form view of an expense.
+After submitting a report, the next step is to wait for the manager to approve it.
 
-All your submitted expense reports can be found in
-:menuselection:`Expenses --> My Expenses --> Expense Reports`.
+.. important::
+   The :ref:`expenses/approve`, :ref:`expenses/post`, and :ref:`expenses/reimburse` sections are
+   **only** for users with the *necessary rights*.
 
+.. _expenses/approve:
 
-How to approve expenses
-=======================
+Approve expenses
+================
 
-HR and team managers get an overview of all expense reports to
-validate from the top menu :menuselection:`To Approve --> Expense Reports to Approve`.
-Such users must have at least *Officers* access rights for *Expenses*.
+In Odoo, not just anyone can approve expense reports, only users with the necessary rights (or
+permissions) can. This means that a user must have at least *Team Approver* rights for the
+*Expenses* app. Employees with the necessary rights can review expense reports, and approve or
+reject them, as well as provide feedback thanks to the integrated communication tool.
 
-.. image:: expenses/approval_01.png
+To see who has rights to approve, go to the main :menuselection:`Settings` app and click on
+:guilabel:`Manage Users`.
+
+.. image:: expenses/manage-users.png
+   :align: center
+   :alt: Check the rights of a user by clicking on Manage Users in the Settings app.
+
+.. Note::
+   If the *Settings* app is not available, then certain rights are not set on the account. In the
+   :guilabel:`Access Rights` tab of a user's card in the :menuselection:`Settings` app, the
+   :guilabel:`Administration` section is set to one of three options:
+
+   - :guilabel:`Blank`: The user cannot access the *Settings* app at all.
+   - :guilabel:`Access Rights`: The user can only view the :guilabel:`User's` section of the
```

Comment:
Yes because it is a section of the settings- part of the UI. Actually, it is "User's & Companies" so I changed it. I tested this again- this is just how 14 works and it is SO weird!